bld/build-namelist

```diff
@@ -2851,6 +2847,14 @@ if ($phys =~ /cam6/ and $nl->get_value('deep_scheme') =~ /ZM/) {
     add_default($nl, 'convproc_wup_max', 'val'=>'4.0D0');
 }
 
+# Aerosol convective processes
+if ($phys =~ /cam_dev/ and $nl->get_value('deep_scheme') =~ /ZM/) {
+    add_default($nl, 'convproc_do_aer', 'val'=>'.true.');
+    add_default($nl, 'convproc_do_evaprain_atonce', 'val'=>'.true.');
+    add_default($nl, 'convproc_pom_spechygro', 'val'=>'0.2D0');
+    add_default($nl, 'convproc_wup_max', 'val'=>'4.0D0');
+}
```

Comment:
Didn't we turn on these convproc for ```cam6``` too?

Reply:
Yes. I put this in so that ZM settings can diverge in `cam_dev`. We may not need it but I think there is already at least one namelist parameter with a different default.